pep-0658.rst

```diff
@@ -0,0 +1,178 @@
+PEP: 658
+Title: Static Distribution Metadata in the Simple Repository API
+Author: Tzu-ping Chung <uranusjr@gmail.com>
+Sponsor:
```

Comment:
@pfmoore here, and in code owners?

Reply:
I’m not sure whether Paul or Donald is better to put in which fields TBH

Reply:
As this is package index stuff, @dstufft is PEP-delegate by default. He can get someone else to handle it if he prefers, and I'd be willing to do it if he wants to pass the job on.

I'd rather not be sponsor, TBH. With the new CODEOWNERS workflow, it looks like that means I'd now be responsible for merging any PRs to the PEP and I don't really have the time to handle that right now...

Reply:
> With the new CODEOWNERS workflow, it looks like that means I'd now be responsible for merging any PRs to the PEP

I'm happy to remain as chief button-pusher to merge uncontroversial PRs :). I think the main point of the CODEOWNERS change is to make it easier for sponsors to stay up to date on changes to their PEPs, but they don't necessarily have to take care of all changes.

Reply:
Donald is gonna be the delegate IIUC, based on the SC's standing delegation for PyPI.

Sponsor is... well, whoever ends up sponsoring this.